resources/guidelines/code/backward-compatibility.md

```diff
@@ -0,0 +1,751 @@
+# Backward Compatibility
+
+## Introduction
+Shopware is a standard eCommerce solution which is used by many customers around the world and is the technical foundation of their online business. As developers of Shopware it is our highest goal to guarantee the reliability of the software for our customers. Always keep in mind that every change you make can have a big impact on the live of our customers, either in a good way, but also in a bad way.
```

Comment:
Blank space before paragraphs is mandatory otherwise, Gitbook will have display issues.